app/models/user.rb

```diff
@@ -15,8 +15,9 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 class User < ActiveRecord::Base
-  devise :database_authenticatable, :recoverable, :rememberable, :trackable, :validatable,
-    :omniauthable, omniauth_providers: [:google_oauth2]
+  devise Rails.application.config.devise_authentication_strategy, :recoverable,
+    :rememberable, :trackable, :validatable,:omniauthable,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space missing after comma.